yarn.lock

```diff
@@ -2,24 +2,24 @@
 # yarn lockfile v1
 
 
-"@algolia/autocomplete-core@1.7.2":
```

Comment:
Are you supposed to change the yarn.lock file too??

Reply:
I did't change it, my yarn was not responding to cross-check my markdown. I had to download a new one